### PR TITLE
Clarifying hit_count is successful runs only

### PR DESCRIPTION
**Pull request by mbbroberg**
Minor clarification to help out cases like #1662.

### CODE REVIEW

File: docs/REST_API.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ Snap task APIs provide the functionality to create, start, stop, remove, enable,
 | deadline                         | task timeout time                       |
 | creation_timestamp               | task creation time                      |
 | last_run_timestamp               | last running time of a task             |
-| hit_count                        | number of times a task ran              |
+| hit_count                        | number of times a task succeeded        |
```
Comment (rashmigottipati) [inline docs/REST_API.md]: Just a suggestion: Would it be better to say "number of times a workflow finished successfully"?

Comment (rashmigottipati) [inline docs/REST_API.md]: Either way, LGTM

Comment (IzabellaRaulin) [inline docs/REST_API.md]: Simpler is better, so it LGTM as it is. @rashmigottipati - are you ok with keeping this as it is? If yes, please merge it. 

@mbbroberg, thank You for your contribution.

**Comment by mbbroberg** [comment]
I like it simpler too. Merging!

**Pull request merged**